pkg/kn/commands/source/apiserver/update_test.go

```diff
@@ -21,13 +21,34 @@ import (
 	"gotest.tools/assert"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
 	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 
 	dynamicfake "knative.dev/client/pkg/dynamic/fake"
 	"knative.dev/client/pkg/sources/v1alpha2"
 	"knative.dev/client/pkg/util"
 )
 
+var (
+	sinkRefsvc1 = duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Kind:       "Service",
+			Name:       "svc1",
+			APIVersion: "serving.knative.dev/v1",
+			Namespace:  "default",
+		},
+	}
+
+	sinkRefsvc2 = duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Kind:       "Service",
+			Name:       "svc2",
+			APIVersion: "serving.knative.dev/v1",
+			Namespace:  "default",
+		},
+	}
```

Comment:
how about function `sinkv1(service, namespace, string)` and use that inline wherever needed ?

(could be done in a follow up)

Reply:
I have added a `createSinkv1` func in `apiserver_test.go`. Turns out this same func exists for binding already as `createServiceSink`. If there is a location to recommend, we could use the same func for both, but let me know what's preferred with regard to that.